docs/wiki/01_STUDY_PARAMETERS_SETUP/02_Clinical_Instruments/07_clinical_test_battery.md

```diff
@@ -0,0 +1,60 @@
+## The Behavioural Test Battery
+
+Now that instruments for the study have been created and installed (if it is not 
+the case, see [Instruments Introduction](01_instrument_intro_prerequisites.md) 
+section), the next step towards data entry into LORIS is to configure the test 
+battery.
+
+The **Test Battery** is the theoretical set of questionnaires administered to a 
+participant at a specific timepoint. The reason we refer to it as "theoretical" 
+in this context is because some exceptions occur in a clinical environment and 
+LORIS does allow for these exception by offering the option to add or remove 
```

Comment:
I don't think we offer a way to remove them? If they're not administered they should just be marked Administration=None.